src/views/create-account-view.ts

```diff
@@ -41,13 +74,31 @@ export class CreateAccountView extends LitElement
         return html`
             <ga-card class="card">
                 <ga-heading>Create an account</ga-heading>
-                <mwc-textfield class="input" outlined required label="User Name"></mwc-textfield>
-                <mwc-textfield class="mail" outlined required label="Mail"></mwc-textfield>
-                <mwc-textfield class="input" outlined required type="password" label="Password"></mwc-textfield>
-                <mwc-textfield class="input" outlined required type="password" label="Confirm password"></mwc-textfield>
-                <mwc-button id="createAccountButton" class="input" raised label="Submit"></mwc-button>
+                <mwc-textfield .value=${this.userName} @change=${this.updateUserName} class="input" outlined required label="User Name"></mwc-textfield>
+                <mwc-textfield .value=${this.userMail} @change=${this.updateUserMail} class="mail" outlined required label="Mail"></mwc-textfield>
+                <mwc-textfield .value=${this.userPass} @change=${this.updateUserPass} class="input" outlined required type="password" label="Password"></mwc-textfield>
+                <mwc-textfield .value=${this.userSecPass} @change=${this.updateUserSecPass} class="input" outlined required type="password" label="Confirm password"></mwc-textfield>
+                <mwc-button id="createAccountButton" @click=${this.createAccount} class="input" raised label="Submit"></mwc-button>
             </ga-card>
             <mwc-button id="loginButton" label="Already have an account?"></mwc-button>
         `;
     }
+
+    private updateUserName(e: any): void
```

Comment:
Use the proper event type here, and the right way to access the input element.
```suggestion
    private updateUserMail(event: Event): void
    {
        this.userMail = (event.currentTarget as HTMLInputElement).value;
    }
```
See:
https://developer.mozilla.org/en-US/docs/Web/API/GlobalEventHandlers/onchange
https://developer.mozilla.org/en-US/docs/Web/API/Event/currentTarget